README.md

```diff
@@ -1,57 +1,32 @@
-# Project Name
-
-(short, 1-3 sentenced, description of the project)
-
-## Features
-
-This project framework provides the following features:
-
-* Feature 1
-* Feature 2
-* ...
-
-## Getting Started
-
-### Prerequisites
-
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
-
-### Installation
-
-(ideally very short)
-
-- npm install [package name]
-- mvn install
-- ...
-
-### Quickstart
-(Add steps to get up and running quickly)
-
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
-
-
-## Demo
-
-A demo app is included to show how to use the project.
-
-To run the demo, follow these steps:
-
-(Add steps to start up the demo)
-
-1.
-2.
-3.
-
-## Resources
-
-(Any additional resources or related projects)
-
-- Link to supporting information
-- Link to similar sample
-- ...
+# Azure Communication Network Traversal sample
+
+### Prerequisites
+
+- Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F). 
+- Install [Node.js](https://nodejs.org/en/download/) Active LTS and Maintenance LTS versions (12.0.0 recommended).
+- Create an Azure Communication Services resource. For details, see [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp). You will need the connection string for this sample.
+- Firefox, Edge or Chrome
+- Computer with a camera
+
+### Installing
+
+- In a Powershell Terminal go to the sample dir and install the following
+
+npm install
+
+## Setup
+
+- Go to the .env (rename sample.env to .env) file of the sample and change the value of COMMUNICATION_CONNECTION_STRING to your connection string (from the Communication Services resource)
+
+## Usage
+
+- Execute: node .\index.js
+- Open your browser, if you are using Mozilla, in the search bar type about:config and search for relay and change it's value to true

```

Comment:
Is there a reason we're referring specifically to Mozilla?
nit: its

Reply:
Yes, for Firefox that's needed but not for the other browsers.

Reply:
Is this a one time thing? If so I'd take it out of the bullet list, and make it a note at the beginning of Usage.
e.g.

## Usage
Note; If you are using Mozilla Firefox, `media.peerconnection.ice.relay_only` needs to be enabled. In the search bar type about:config. Search for `media.peerconnection.ice.relay_only` and change its value to true.